site/docs/transform/window.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: docs
+menu: docs
+title: Window
+permalink: /docs/window.html
+---
+
+The window transform performs calculations over sorted groups of data objects. These calculations including ranking, lead/lag analysis, and aggregates such as running sums and averages. Calculated values are written back to the input data stream.
+
+## Window Field Definition
+
+{: .suppress-error}
+```json
+// A View Specification
+{
+  ...
+  "transform": [
+    {
+      // Window Transform
+      "window": [{
+          "op": "...",
+          "field": "...",
+          "as": "..."
+      }],
+      "groupby": [
+          "..."
+      ],
+      "frame": "..."
+    }
+     ...
+  ],
+  ...
+}
+```
+
+## Window Transform Definition
+
+| Property    | Type | Description  |
+| :----------- | :-------: | :------------|
+| window | WindowFieldDef[] | The definition of the fields in the window, and what calculations to use. |
```

Comment:
Use a special tag and make sure that you have the JSDocs in the code include comments for all of these 

`{% include table.html props="window,frame,ignorePeers,groupBy,sort" source="WindowTransform" %}`


(Window Only Operation Reference can't use this table thing though as it's a list of valid values, not property table.)

Reply:
Oh thats pretty cool. I didn't know about that tag. Just updated it to use that for the 3 object definitions (WindowTransform, WindowSortField, WIndowFieldDef).